### DISCUSSION TITLE
Cannot run program "yq"

### DISCUSSION THREAD

**Discussion by zhaorendong** [strimzi/strimzi-kafka-operator#4705] [General]
[ERROR] Command execution failed.
java.io.IOException: Cannot run program "yq" (in directory "C:\idea-workspace\strimzi-kafka-operator\test-container"): CreateProcess error=2, 系统找不到指定的文件。

**Comment by scholzj** [comment]
And do you have it installed it? There is list of prerequisites in the developer guide: https://github.com/strimzi/strimzi-kafka-operator/blob/main/development-docs/DEV_GUIDE.md#build-pre-requisites
